packages/test/test-end-to-end-tests/src/test/deRehydrateContainerTests.spec.ts

```diff
@@ -565,7 +565,7 @@ describeFullCompat(`Dehydrate Rehydrate Container Test`, (getTestObjectProvider)
             // Create and reference another dataStore
             const { peerDataStore: dataStore2 } = await createPeerDataStore(defaultDataStore.context.containerRuntime);
             defaultDataStore.root.set("dataStore2", dataStore2.handle);
-            //* Unnecessary?
```

Comment:
these changes don't look necessary

Reply:
Ironic, huh.  It was part of the revert.  But I'll skip it for this branch.